Makefile

```diff
@@ -1,8 +1,30 @@
 # Use bash as shell
 SHELL = /bin/bash
 
+# Check for dirty :D 
+define check_dirty
```

Comment:
why not create `check-git-dirty.sh` like in https://github.com/Kuadrant/limitador-operator/blob/main/utils/check-git-dirty.sh?

Reply:
I think that's a good suggestion, @eguzki ! Will seek to implement.